model/riscv_insts_zcmp.sail

```diff
@@ -0,0 +1,643 @@
+
+function clause execute (CM_PUSH(rlist, sp54)) = {
+  var bytes : bits(4) = zeros();
+
+  if (sizeof(xlen) == 32) then {
+    bytes = 0x4
+  };
+
+  if (sizeof(xlen) == 64) then {
+    bytes = 0x8
+  };
+
+  reg_list : vector (13,dec, bits(5)) = [0b00000,0b00000,0b00000,0b00000,0b00000,0b00000,0b00000,0b00000,0b00000,0b00000,0b00000,0b00000,0b00000];
+
+  match rlist {
+    0x4  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00001],
+    0x5  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b01000, 0b00001],
+    0x6  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b01001, 0b01000, 0b00001],
+    0x7  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b10010, 0b01001, 0b01000, 0b00001],
+    0x8  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+    0x9  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b10100, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+    0xa  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b10101, 0b10100, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+    0xb  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b00000, 0b10110, 0b10101, 0b10100, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+    0xc  => reg_list = [0b00000, 0b00000, 0b00000, 0b00000, 0b10111, 0b10110, 0b10101, 0b10100, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+    0xd  => reg_list = [0b00000, 0b00000, 0b00000, 0b11000, 0b10111, 0b10110, 0b10101, 0b10100, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+    0xe  => reg_list = [0b00000, 0b00000, 0b11001, 0b11000, 0b10111, 0b10110, 0b10101, 0b10100, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+    0xf  => reg_list = [0b11011, 0b11010, 0b11001, 0b11000, 0b10111, 0b10110, 0b10101, 0b10100, 0b10011, 0b10010, 0b01001, 0b01000, 0b00001],
+  };
+
+
+  var stack_adj_base : bits(xlen) = zeros();
+  var stack_adj : bits(xlen) = zeros();
+
+  if (sizeof(xlen) == 32) then {
+    match rlist {
+      0x4 => {stack_adj_base = zero_extend(0x10);},
+      0x5 => {stack_adj_base = zero_extend(0x10);}, 
+      0x6 => {stack_adj_base = zero_extend(0x10);},
+      0x7 => {stack_adj_base = zero_extend(0x10);},
+      0x8 => {stack_adj_base = zero_extend(0x20);},
+      0x9 => {stack_adj_base = zero_extend(0x20);},
+      0xa => {stack_adj_base = zero_extend(0x20);},
+      0xb => {stack_adj_base = zero_extend(0x20);}, 
+      0xc => {stack_adj_base = zero_extend(0x30);}, 
+      0xd => {stack_adj_base = zero_extend(0x30);}, 
+      0xe => {stack_adj_base = zero_extend(0x30);}, 
+      0xf => {stack_adj_base = zero_extend(0x40);},
+    };
+    } else {
```

Comment:
Another formatting mistake. Reviewing it is tedious, please just take care in what you do.